app/src/main/java/com/randomappsinc/studentpicker/grouping/GroupingMakingFragment.java

```diff
@@ -48,7 +48,6 @@ public static GroupingMakingFragment getInstance(String listName) {
     private String listName;
     private DataSource dataSource;
```

Comment:
We can remove DataSource too right?

Reply:
@Gear61  you mean to make it as a local variable

Reply:
![image](https://user-images.githubusercontent.com/25705658/73384032-2127a380-42d3-11ea-97e7-33fa97f3c8df.png)